projects/picker/src/lib/date-time/calendar.component.ts

```diff
@@ -347,6 +358,16 @@ export class OwlCalendarComponent<T> implements AfterContentInit, AfterViewCheck
     this.pickerMomentChange.emit(this.pickerMoment);
   }
 
+  public jumpToToday(): void {
+    const now = this.dateTimeAdapter.now();
+    let today = this.dateTimeAdapter.setHours(now, 0);
+    today = this.dateTimeAdapter.setMinutes(today, 0);
+    today = this.dateTimeAdapter.setSeconds(today, 0);
```

Comment:
Didn't we only want to do if the time UI is not visible

- pickerType = 'both' => timer UI visible => set the time to now
- pickerType = 'calendar' => timer UI not visible =>  set the time to 0
- pickerType = 'timer' => today button not visible

Reply:
Yeah but it was more complicated than expected because 
```
const now = this.dateTimeAdapter.now();

this.handlePickerMomentChange(now);
this.dateSelected(now);
```
If the picker has already been interacted with does not change back to "now" time. As a result I was even able to select To dates that were chronologically after From dates, because the check for this condition does not factor in the time of the day.
I can see if I can meddle with the time-picker in the view and set it manually but I am not sure at that point if it is worth it based on what we want to do with it.

Reply:
If that's not easily possible then sure. keep it as it is right now 👍🏼